script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -747,57 +801,64 @@ The bayonet that's been thrust into my chest is trembling.
 [sha:9fa01da08215b4c81e35381602e66dc50c1016be]{
 -- Page 433, Offset 34532.
 -- 騙して、裏切って、こうして残酷に追い詰めているのに、なんでそんな穏やかな顔をしてるんですか、貴方は……！」
-I deceived and betrayed you, and now I'm mercilessly hunting you down, so why do you look so calm...?!"
+I deceived you,
+ I betrayed you,
+ and I mercilessly hunted you down and cornered you, so why...
+ why do you look so damn calm...?!"
 }
 [sha:7681e1cde97a02affdc16442862bb6628a642580]{
 -- Page 434, Offset 34533.
 -- かつん、と。
-%{i}Step%{/i}.
+%{i}Tap%{/i}.
 }
 [sha:050a317b0980e715260cbbd947fc55e181dbc5ae]{
 -- Page 434, Offset 34534.
 -- 銃口はそのままで、先輩は俺に踏みこんできた。
-With the bayonet still lodged in my chest, Senpai takes another step forward.
+Without moving her weapon, Senpai takes another step forward.
 }
 [sha:faffb8f71f1ec09fd02255b700ade7290a8e964c]{
 -- Page 435, Offset 34535.
 -- 「答えなさい……！　わたしは貴方を殺すんです。貴方の都合なんて関係なく、ただ一方的に殺そうとしているんです……！
-"Answer me...! I'm going to kill you――murder you with no regard for your well-being whatsoever...!
+"Answer me...!
+ I'm going to kill you.
+ I'm going to kill you whether you like it or not, just to satisfy my own ends...!
 }
 [sha:24a83e481f51427cc3103b8fd2ab813e24c3cedc]{
 -- Page 435, Offset 34536.
 -- ならせめて、わたしを恨まないと貴方は報われないじゃないですか……！」
-There's nothing to gain by not hating me!"
+What could you possibly have to gain by not hating me?!"
 }
 [sha:594ee44995ebc3c08c220a929eeb7b1fb68b24ef]{
 -- Page 436, Offset 34537.
 -- 火のついた激しさで、先輩は問い詰めてくる。
-Senpai questions me in a fiery rage.
+Anger burns in Senpai's voice as she demands an answer.
```

Comment:
Blazing intensity as a noun doesn't really feel right here. Prefer original structure overall as well.

Reply:
Well, my problem is probably just that "as she demands an answer" because the "as" doesn't connect that well.
If you don't like "blazing intensity", I think at least this is better.
>In demand of an asnwer, Senpai's voice burns with anger.